lib/google_scraping_web/router.ex

```diff
@@ -61,4 +64,25 @@ defmodule GoogleScrapingWeb.Router do
       forward "/mailbox", Plug.Swoosh.MailboxPreview
     end
   end
+
+  ## Authentication routes
+
+  scope "/", GoogleScrapingWeb do
+    pipe_through [:browser, :redirect_if_user_is_authenticated]
+
+    get "/users/register", UserRegistrationController, :new
+    post "/users/register", UserRegistrationController, :create
```

Comment:
We can do something cool with Phoenix routing where we can use `resources` instead so we wouldn't need to manually create routes repeatedly.

More information on what routes it covers [here](https://hexdocs.pm/phoenix/routing.html#resources) 😄  

```suggestion
    resources "/users/register", UserRegistrationController, only: [:new, :create]
```

We can also do this with the rest of the routes here 👍